### DISCUSSION TITLE
Using absolute path for comsol-server on dedicated server

### DISCUSSION THREAD

**Discussion by louisreg** [MPh-py/MPh#61] [General]
Hi, 
First of all many thanks for MPh-py. This is a great tool and it is very useful for us. 
We managed to get MPh-py running without any issue on our laptop. However, we tried to get it running on a dedicated linux server but we're having some issues because MPh-py does not find comsol-server. This is mainly because the installation directory is located in a weird location but we can't do anything about that. We managed to get it working by modifying your code and by writing the absolute path of comsol-server in your code. However this is not very convenient, especially if you release new versions of this code. 
Is it possible to put the absolute path as an argument of a function instead of letting the code finding itself? If not, would it be possible to add this feature? that would be very useful for us!
Many thanks in advance
best
Louis

**Comment by john-hen** [comment]
Hi Louis. Can you convert this Discussion to an Issue? You should see a "Create issue from discussion" link on the right. I could do it for you, but it makes more sense if the post is in your name. We'll still have to "discuss" it, true. I have a couple of questions. But this is effectively a feature request, and that's something an "issue" is designed to track.
